src/content/plugins/split-chunks-plugin.mdx

```diff
@@ -230,7 +230,7 @@ When the chunk has a name already, each part will get a new name derived from th
 
 `maxSize` option is intended to be used with HTTP/2 and long term caching. It increases the request count for better caching. It could also be used to decrease the file size for faster rebuilding.
 
-T> `maxSize` takes higher priority than `maxInitialRequest/maxAsyncRequests`. Actual priority is `maxInitialRequest/maxAsyncRequests < maxSize < minSize`.
+T> `maxInitialRequest/maxAsyncRequests` takes higher priority than `maxSize`. Actual priority is `maxSize < maxInitialRequest/maxAsyncRequests < minSize`.
 
```

Comment:
https://github.com/webpack/webpack.js.org/issues/7118#issuecomment-1912975948

Did you verify the priority locally?

Reply:
Just verified locally. https://github.com/haijie-x/webpack-split-chunks-demo
You can clone this repo, and then 
```bash
node run.js --single=maxRequest>maxSize
``` 
this means the options
```javascript
```

Reply:
Just verified locally. 

```
1. clone repo https://github.com/haijie-x/webpack-split-chunks-demo
2. node ./run.js --single=maxRequest-maxSize-priority
```
--single=maxRequest-maxSize-priority means that the options is following:
```
splitChunks: {
    cacheGroups: {
      BigFile: {
        test: /30KB/,
        priority: 10,
        maxAsyncRequests: 1,
        maxSize: 40 * 1024
      }
    }
}
```
when without `maxAsyncRequests: 1` option
```
cacheGroups: {
      BigFile: {
        test: /30KB/,
        priority: 10,
        // maxAsyncRequests: 1,
        maxSize: 40 * 1024
      }
    }
```
after build, bundle is following
```
BigFile~async-a~59f91fb7.043726bc102ae705ae9e.chunk.js 
BigFile~async-a~d0b264a5.dcdbd653939c854f5b77.chunk.js 
main.5f57ac6a0fe9ebb45415.js
BigFile~async-a~a22135d7.30009d898c87440a1e1d.chunk.js 
async-a.f0391a3373ef595394f3.chunk.js
```
when with `maxAsyncRequests: 1` option

after build, bundle is following
```
async-a.707f70c4a34ad633be9f.chunk.js        
main.4a5d4486260850356d35.js                  
vendors~async-a.79271a167b280cd94ef1.chunk.js
```

Obviously, `maxAsyncRequests` gets higher priority. and Webpack will not try to split one large vendor chunk into three `maxSize < 40KB` chunks.